### ISSUE TITLE
ValueError: Invalid color value

### ISSUE THREAD

**Issue by sander76** [xhtml2pdf/xhtml2pdf#352]
Running python 2.7 and the pre release of xhtml2pdf (couldn't get the latest running).
On the cli I use:

`xhtml2pdf https://ww.google.com`

I get a `ValueError`

```bash
Converting www.google.com to C:\virtualenvs\xhtml\Scripts\www.google.com.pdf...
Traceback (most recent call last):
  File "c:\python27\Lib\runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "c:\python27\Lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "C:\virtualenvs\xhtml\Scripts\xhtml2pdf.exe\__main__.py", line 9, in <module>
  File "c:\virtualenvs\xhtml\lib\site-packages\xhtml2pdf\pisa.py", line 153, in command
    execute()
  File "c:\virtualenvs\xhtml\lib\site-packages\xhtml2pdf\pisa.py", line 374, in execute
    xml_output=xml_output,
  File "c:\virtualenvs\xhtml\lib\site-packages\xhtml2pdf\document.py", line 97, in pisaDocument
    encoding, context=context, xml_output=xml_output)
  File "c:\virtualenvs\xhtml\lib\site-packages\xhtml2pdf\document.py", line 59, in pisaStory
    pisaParser(src, context, default_css, xhtml, encoding, xml_output)
  File "c:\virtualenvs\xhtml\lib\site-packages\xhtml2pdf\parser.py", line 708, in pisaParser
    pisaLoop(document, context)
  File "c:\virtualenvs\xhtml\lib\site-packages\xhtml2pdf\parser.py", line 648, in pisaLoop
    pisaLoop(node, context, path, **kw)
  File "c:\virtualenvs\xhtml\lib\site-packages\xhtml2pdf\parser.py", line 593, in pisaLoop
    pisaLoop(nnode, context, path, **kw)
  File "c:\virtualenvs\xhtml\lib\site-packages\xhtml2pdf\parser.py", line 593, in pisaLoop
    pisaLoop(nnode, context, path, **kw)
  File "c:\virtualenvs\xhtml\lib\site-packages\xhtml2pdf\parser.py", line 593, in pisaLoop
    pisaLoop(nnode, context, path, **kw)
  File "c:\virtualenvs\xhtml\lib\site-packages\xhtml2pdf\parser.py", line 593, in pisaLoop
    pisaLoop(nnode, context, path, **kw)
  File "c:\virtualenvs\xhtml\lib\site-packages\xhtml2pdf\parser.py", line 593, in pisaLoop
    pisaLoop(nnode, context, path, **kw)
  File "c:\virtualenvs\xhtml\lib\site-packages\xhtml2pdf\parser.py", line 537, in pisaLoop
    CSS2Frag(context, kw, isBlock)
  File "c:\virtualenvs\xhtml\lib\site-packages\xhtml2pdf\parser.py", line 279, in CSS2Frag
    c.frag.backColor = getColor(c.cssAttr["background-color"])
  File "c:\virtualenvs\xhtml\lib\site-packages\xhtml2pdf\util.py", line 98, in __call__
    res = self.func(*args, **kwargs)
  File "c:\virtualenvs\xhtml\lib\site-packages\xhtml2pdf\util.py", line 152, in getColor
    return toColor(value, default)  # Calling the reportlab function
  File "c:\virtualenvs\xhtml\lib\site-packages\reportlab\lib\colors.py", line 870, in __call__
    raise ValueError('Invalid color value %r' % arg)
ValueError: Invalid color value '/images/branding/googlelogo/1x/googlelogo_white_background_color_272x92dp.png'
```

**Comment by cptx032** [comment]
Hi, I'm getting the same error. I've get a large HTML to convert, inside it ive found something like:

```html
<span style='color: var(--neutralLight)'></span>
```

My opinion is that xhtml2pdf in this case don't need to raise an error, to act more like a browser-behavior-way-of-life.

**Comment by JordanMurphy1799** [comment]
I'm also getting the same error. I have a set color palette incorporated into my stylesheet.

`Invalid color value '<css function: var(--blue)>`

I'm not sure how to get around this without trying to catch the error or add redundancy to the CSS by hard-coding those color values.

**Comment by M-Tuladhar-reddy345** [comment]
I faced the same issue
But reinstalling the latest version helped me.

**Comment by timoludwig** [comment]
This was fixed in #594
